source/FreeplayState.hx

```diff
@@ -271,7 +274,10 @@ class FreeplayState extends MusicBeatState
 		#end
 
 		#if PRELOAD_ALL
-		FlxG.sound.playMusic(Paths.inst(songs[curSelected].songName), 0);
+		songWait.cancel();
```

Comment:
If you want to have the music pause between each selection:
```suggestion
		FlxG.sound.music.stop();
		songWait.cancel();
```

Reply:
(sorry for the delayed response)
i think letting the previous selected song play while scrolling is nicer, but that's just me.
i'll leave the decision to nm99 if he merges